qutebrowser/browser/webengine/interceptor.py

```diff
@@ -71,6 +115,10 @@ def interceptRequest(self, info):
 
         url = info.requestUrl()
         first_party = info.firstPartyUrl()
+        # Per QWebEngineUrlRequestInfo::ResourceType documentation, if we fail
+        # our lookup, we should fall back to ResourceTypeUnknown
+        resource_type = RequestInterceptor.QWebEngineResourceTypeMap.get(
+            info.resourceType(), interceptors.ResourceType.unknown)
```

Comment:
IMHO we should at least log unknown (as in "not in the dict") resource types as a warning.